.github/workflows/cross-post.yml

```
@@ -0,0 +1,95 @@
+        if: ${{ github.event.inputs.platforms == 'all' || github.event.inputs.platforms == 'hashnode' || github.event.inputs.platforms == '' }}
+        env:
+          HASHNODE_API_TOKEN: ${{ secrets.HASHNODE_API_TOKEN }}
+          HASHNODE_PUBLICATION_ID: ${{ secrets.HASHNODE_PUBLICATION_ID }}
+        run: |
+          if [ -n "$HASHNODE_API_TOKEN" ] && [ -n "$HASHNODE_PUBLICATION_ID" ]; then
+            echo "Publishing to Hashnode..."
+            tsx scripts/publish-hashnode.ts "${{ github.event.inputs.post_slug || 'auto' }}" "${{ github.event.inputs.canonical_url || 'https://mac-portfolio.vercel.app' }}"
+          else
+            echo "Skipping Hashnode - missing credentials"
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)